### DISCUSSION TITLE
i2pd.conf bandwidth setting

### DISCUSSION THREAD

**Discussion by Bob-F-xxx** [PurpleI2P/i2pd#1843] [Q&A] [answered]
**bandwidth = P**  and  **bandwidth = X** are clear but what is the right syntax if I want to use real numbers?  Is just the value in KB/s or should I add units too?

**Comment by r4sas** [comment, marked as answer]
https://i2pd.readthedocs.io/en/latest/user-guide/configuration/#general-options

You can use letters and integers. If you set integer value, router will publish flag with higher bandwidth level, but your value will be preferred.
Example: You set `bandwidth = 2000`, router will publish flag `P` (limit 2048) but will limit building tunnels at 2000 KBps.

**Comment by Bob-F-xxx** [comment]
I did read that but the example in the config explains it in a weird way.  I mean, 2048**KBs/sec** is not exactly a common notation.  but I know now.  Thanks.
